### PR TITLE
[#4] Buttons

### LINKED ISSUE #4: Buttons
Implement styles for buttons. See the design style guide in Figma.

Reference:
buttons

### PR DESCRIPTION
**Pull request by LavanyaThuyamani**
Implements the styles for Buttons with help of the design style guide in Figma.

### CODE REVIEW

File: css/src/_base.scss [hunk 1]
```diff
@@ -68,6 +68,14 @@ a {
...
 }
 
+button {
+    border: none;
+    border-radius: 6px;
+    padding: 10px 15px;
+    margin: 0 20px 44px 0;
+    font-weight: bold;
+}
+
```
Comment (mageshravi) [inline css/src/_base.scss]: Not all buttons will follow this style. These styles should be defined within the `.m-btn` module.

File: css/src/_m-btn.scss [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+        color: var(--body-txt);
+
+        &--primary {
+            --btn-bg: #528245;
+            --btn-bg--hover: #79bd66;
+            --btn-bg--active: #35552c;
+            --btn-box-shadow--hover: rgba(77, 121, 65, 0.6);
+
+            color: var(--white-txt);
```
Comment (mageshravi) [inline css/src/_m-btn.scss:85]: This property is not required. Already defined in line 26.

File: css/src/_m-btn.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        &--secondary {
+            --btn-bg: #4f7c8e;
+            --btn-bg--hover: #8bb5c5;
+            --btn-bg--active: #3b5d6b;
+            --btn-box-shadow--hover: rgba(82, 107, 117, 0.6);
+
+            color: var(--white-txt);
```
Comment (mageshravi) [inline css/src/_m-btn.scss:94]: Property not required. Already defined in light theme.

File: css/src/_m-btn.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        &--danger {
+            --btn-bg: #af2e3d;
+            --btn-bg--hover: #e46d7b;
+            --btn-bg--active: #84222e;
+            --btn-box-shadow--hover: rgba(138, 64, 73, 0.6);
+
+            color: var(--white-txt);
```
Comment (mageshravi) [inline css/src/_m-btn.scss:103]: Not required. already defined in light theme.

File: css/src/_m-btn.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        &--warning {
+            --btn-bg: #c18d0a;
+            --btn-bg--hover: #f4c54d;
+            --btn-bg--active: #916a08;
+            --btn-box-shadow--hover: rgba(125, 101, 38, 0.6);
+
+            color: var(--white-txt);
```
Comment (mageshravi) [inline css/src/_m-btn.scss:112]: not required. already defined in light theme.

File: index.html [hunk 6]
```diff
@@ -48,6 +48,32 @@ <h6>Heading 6</h6>
...
+    <div>
+        <button class="m-btn m-btn--compact">Button</button>
+        <button class="m-btn m-btn--primary m-btn--compact">Button</button>
+        <button class="m-btn m-btn--secondary m-btn--compact">Button</button>
+        <button class="m-btn m-btn--danger m-btn--compact">Button</button>
+        <button class="m-btn m-btn--warning m-btn--compact">Button</button>
+    </div>
 </body>
 
 </html>
```
Comment (mageshravi) [inline index.html]: Add an empty line at the end.

**Review by mageshravi — CHANGES_REQUESTED** [review]
Changes required.

File: css/src/_m-btn.scss (hunk 2 above)
Comment (LavanyaThuyamani) [reply css/src/_m-btn.scss:85]: This property is required.

File: css/src/_m-btn.scss (hunk 3 above)
Comment (LavanyaThuyamani) [reply css/src/_m-btn.scss:94]: This property is required.

File: css/src/_m-btn.scss (hunk 4 above)
Comment (LavanyaThuyamani) [reply css/src/_m-btn.scss:103]: This property is required.

File: css/src/_m-btn.scss (hunk 5 above)
Comment (LavanyaThuyamani) [reply css/src/_m-btn.scss:112]: This property is required.

**Review by mageshravi — APPROVED** [review]
Changes look good.

**Pull request merged**